content/administration/install/deploy.rst

```diff
@@ -179,21 +179,36 @@ or 'verify-full'
 Builtin server
 ==============
 
-Odoo includes built-in HTTP servers, using either multithreading or
-multiprocessing.
+Odoo includes built-in HTTP, cron and live-chat servers, using either
+multi-threading or multi-processing.
+
+The **multi-threaded** server is a simpler server primarely used for
+development, demonstrations and for compatibility with various operation
+systems (including Windows). A new thread is spawned for every new HTTP
+request, even for long running requests such as longpolling. Extra daemonic
+cron threads are spawned too. Due to a Python limitation (GIL), it doesn't make
+the best use of the hardware.
+
+The multi-threaded server is the default server, also for docker containers, it
+is selected by leaving the :option:`--workers <odoo-bin --workers>` option out
+or to set it to ``0``.
```

Comment:
```suggestion
The multi-threaded server is the default server. For docker containers, it
is selected by leaving the :option:`--workers <odoo-bin --workers>` option out
or setting it to ``0``.
```

Reply:
> The multi-threaded server is the default server, also for docker containers. It
is selected by leaving the :option:`--workers <odoo-bin --workers>` option out
or setting it to ``0``.